### ISSUE TITLE
GitLab repo without README fails unexpectedly

### ISSUE THREAD

**Issue by sverhoeven** [fair-software/howfairis#65]
Checking https://gitlab.com/a.vijaykumar/bicgstab_l returns an error, I expected it to give `● ○ ○ ○ ○`.
```
howfairis https://gitlab.com/a.vijaykumar/bicgstab_l
Checking compliance with fair-software.eu...
url: https://gitlab.com/a.vijaykumar/bicgstab_l
Did not find a README[.md|.rst] file at https://gitlab.com/a.vijaykumar/bicgstab_l/-/raw/master/
(1/5) repository
      × has_open_repository
(2/5) license
AttributeError: 'NoneType' object has no attribute 'find'
```

**Comment by sverhoeven** [comment]
I expect it is due to absent README.* in the repo.

**Comment by jspaaks** [comment]
This is indeed incorrect behavior

**Comment by eriktks** [comment]
solved by PR #111
